app/models/suggested_councillor.rb

```diff
@@ -2,4 +2,6 @@ class SuggestedCouncillor < ActiveRecord::Base
   has_one :authority, through: :councillor_contribution
   belongs_to :councillor_contribution
   validates :councillor_contribution, :name, :email, presence: true
+  validates_email_format_of :email, :message => "format seems not right.
+  Does it follow the format of example@email.com?", :unless => Proc.new{|c| c.email.blank?}
```

Comment:
Does this `unless` option mean that this wont be passed as a message if the email is blank? Reading the reason for this on the [commit message](https://github.com/openaustralia/planningalerts/pull/1206/commits/4d6551b9c451e0189e4fd7e0bee72d8ae4f755fb):

> To display different error messages when the format is invalid and when email is blank

I'm interested in the thinking behind this @hisayohorie . What was happening before this change?

This isn't the right place to implement this behavior. It's a good hint that the commit message for this is about changing the 'display'. The model validations should focus on getting valid data to database/model. The display of error messages should be handled in the view and with helper methods (possibly on the model).